fredix1/React-practise
# #161686039 As a user I should see a list of meals

#### What does this PR do?

 - Enables a user see a list of available meals

#### Description of Task to be completed?

 - Ensure the application can fetch meals from the The Meal Db Api
- Ensure a user can see all the available meals
#### How should this be manually tested?
- clone the Repository `https://github.com/fredix1/React-practise.git`
 checkout into the branch `ft-see-list-of-meals-161686039`
- install dependencies `yarn install`
- Run the application `yarn start`
- Once the application loads you should see a list of meals
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161686039](https://www.pivotaltracker.com/story/show/161686039)
#### Screenshots (if appropriate)
N/A

#### Questions:
